students/17th/team3/jaeyiahn/user/views.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models import User
+
+#회원가입
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data     = json.loads(request.body)
+
+            username = data.get('username')
+            email    = data.get('email')
+            phone    = data.get('phone')
+            password = data.get('password')
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message': '중복된 email'}, status=400)
+            
+            if '@' and '.' not in email:
+                return JsonResponse({'message': '잘못된 email형식'}, status=400)
+            
+            if len(password) < 8:
```

Comment:
```suggestion
            if len(password) < MINIMUM_PASSWORD_LENGTH:
```
위와 같이 8이라는 값은 하드코딩하기보다, 상수로 작성해서 관리하는게 좋습니다. 
향후 기준이 달라질 경우 상수 값만 변경하면 되기 떄문입니다.
상수의 선언은 코드 상단 import 아래에 선언해주세요.

Reply:
네 반영하여 적용하도록 하겠습니다